### ISSUE TITLE
add a JavaScript API so that the module can be used from other script

### ISSUE THREAD

**Issue by MegaJSDeveloper** [catdad/electronmon#42]
How i can run and force reload electronmon from gulp task?
Electronmon have JS API?

like:
```javascript
var electronmon = require('electronmon')

var app = electronmon.start('./main.js')
app.reload()
app.close()
app.on('reload', function(){ })
// e.t.c.
```

**Comment by catdad** [comment]
I must have missed this one, sorry for the delay.

Officially, this module does not have an API, and this is intentional. Can you tell me more about your use case for forcing a reload? Do you mean to do this when gulp detects a file change? I am not sure what the benefit of this is, seeing as gulp should write new source files to disk and electronmon will then detect those changes. Are you meaning to reload the application when non-code files are changed, or when supplementary files used at runtime are changes in a different directory?

**Comment by MegaJSDeveloper** [comment]
@catdad happy new year to you, friend! I generally gave a hint that it would not hurt to make a NodeJS api, this will allow people to write all kinds of plugins on gulp and webpack using your library)

**Comment by catdad** [comment]
There is now an API available. If you do find a use for it and find things missing, feel free to file an issue for them.

Published in 1.0.0
